tests/client-dbus/requirements.txt

```diff
@@ -1,3 +1,3 @@
 tox==3.2.1
-black==18.9b0
+black==18.3b0
```

Comment:
I think this is supposed to be 19 not 18

Reply:
Whoops! Fixed.